README.md

```diff
@@ -25,6 +25,7 @@ Spread
 [LXD backend](#lxd)  
 [QEMU backend](#qemu)  
 [Google backend](#google)  
+[Openstack backend](#openstack)  
```

Comment:
The casing of the project name is incorrect, both in documentation and in the code itself. The project seems to call itself as _OpenStack_ instead of _OpenStack,_ so I would suggest we follow suit and do a `s/Openstack/OpenStack/` everywhere. Let's please keep _"openstack",_ though (not _"openStack"_).